projects/ngneat/reactive-forms/src/lib/formControl.spec.ts

```diff
@@ -14,6 +15,48 @@ const validatorExample = new FormControl<string, NgValidatorsErrors>('', {
   }
 });
 
+describe('FormControl valueChanges$ diff() operator', () => {
+  const control = new FormControl<string>();
+  const spy = jest.fn();
+  control.value$.pipe(diff()).subscribe(spy);
+
+  it('should be initialized', () => {
+    expect(spy).toHaveBeenCalledWith(null);
+    expect(spy).toHaveBeenCalledTimes(1);
+  });
+
+  it('should filter duplicated calls', () => {
+    control.patchValue('patched');
+    expect(spy).toHaveBeenCalledWith('patched');
+    control.patchValue('patched');
```

Comment:
It'll be more clear if you add ` expect(spy).toHaveBeenCalledTimes(2)` before you call `patchValue`

Reply:
Thanks, I will add it.